docs/docs/examples/qa-automation.md

```diff
@@ -0,0 +1,223 @@
+# QA Automation with LaVague and Pytest
```

Comment:
There should bre some CI to test the code on the .md. Because it might break at any new commit.
You can take inspiration from what @lyie28  did in the test-docs-code workflow

Reply:
Code on the `.md` is just a subset of the script, can it be tested as is ?

Reply:
Since the 2 are not automatically synchronized, one might change and not the other. But we might just put that back for later. It still should sit on the back of our mind though.

Reply:
Sounds good!